vita3k/gui/src/content_manager.cpp

```diff
@@ -103,7 +103,7 @@ static void get_save_data_list(GuiState &gui, EmuEnvState &emuenv) {
 
     for (const auto &save : fs::directory_iterator(SAVE_PATH)) {
         const auto title_id = save.path().stem().generic_string();
-        if (fs::is_directory(save.path()) && !fs::is_empty(save.path()) && get_app_index(gui, title_id) != gui.app_selector.user_apps.end()) {
+        if (fs::is_directory(save.path()) && !fs::is_empty(save.path()) && get_app_index(gui, title_id) != nullptr) {
```

Comment:
```suggestion
        if (fs::is_directory(save.path()) && !fs::is_empty(save.path()) && get_app_index(gui, title_id)) {
```

Here like this should works too no ? You using this for title 🤔

Reply:
Yes, this also works, but I feel like the != nullptr is a bit more expressive (you get to know that get_app_index returns a pointer and not a boolean). Although that's only personal preference.

Reply:
Vita3k more no using it, dev have some time refactor for remove all of this with compare on nullptr 
Is more clean